src/App.tsx

```diff
@@ -8,13 +8,18 @@ import { Element } from './types/Common';
 const App = () => {
   const [elements, setElements] = useState<Element[]>([]);
 
-  const hendleCreateElement = (value: Element) => {
-    setElements((prev) => [...prev, value]);
+  const handleCreateElement = (value: Element) => {
+    setElements((prev) => {
+      // uuid = latest/biger prev id +1
+      const uuid =
+        prev.length > 0 ? (+prev[prev.length - 1].id + 1).toString() : '1';
```

Comment:
a very simple unique id can be a global variable that's incremented with every new object
```js
let id = 0;

const handleCreateElement = () => {
  const eid = id++;
}